shardingsphere-jdbc/shardingsphere-jdbc-orchestration/src/main/java/org/apache/shardingsphere/driver/orchestration/internal/datasource/OrchestrationShardingSphereDataSource.java

```diff
@@ -69,7 +69,7 @@
     private ShardingSphereDataSource dataSource;
     
     public OrchestrationShardingSphereDataSource(final OrchestrationConfiguration orchestrationConfig) throws SQLException {
-        super(new ShardingOrchestrationFacade(orchestrationConfig, Collections.singletonList(DefaultSchema.LOGIC_NAME)));
+        super(new ShardingOrchestrationFacade(orchestrationConfig, Collections.singletonList(DefaultSchema.LOGIC_NAME), 3307));
```

Comment:
`OrchestrationShardingSphereDataSource` is provided for `sharding-jdbc`, can not specify a fixed port , `sharding-jdbc` should still retain threadId as the unique identifier of the instance.

Reply:
Agreed! This looks to be a mistake at my end